src/js/WebEngineAppContainer.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default class WebEngineAppContainer extends React.Component {
+    render() {
+        return (<div className="webengine-app-container">
+            <iframe src={this.props.iframeUrl} style={{ display: 'none' }} />
+        </div>);
+    }
+}
```

Comment:
new line

Reply:
added in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/b79ab73fcfd760a30437c25d53d1c6665ee29526